bin/build.js

```diff
@@ -73,7 +78,8 @@ const buildOptions = {
   serviceName: argv.s,
   zip: argv.z,
   tsconfig: argv.t,
-  enableRuntimeSourceMaps: argv['enable-runtime-source-maps']
+  enableRuntimeSourceMaps: argv['enable-runtime-source-maps'],
+  cacheDirectory: argv.c
```

Comment:
`argv['enable-cache-directory']` would have been more consistent and readable in my opinion. I am not a huge fan of short argument names in most situations (they are not self-documenting).

Reply:
Fixed. Thanks for the feedback!